### PR TITLE
Remove temp task image directory after import

### PR DESCRIPTION
**Pull request by gregarndt**

### CODE REVIEW

**Comment by walac** [comment]
r+

File: test/integration/allow_ptrace_test.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ import Debug from 'debug';
 
 let debug = Debug('docker-worker:test:allow-ptrace-test');
 
-suite('allowPtrace feature', () => {
+suite.skip('allowPtrace feature', () => {
```
Comment (djmitche) [inline test/integration/allow_ptrace_test.js]: unrelated?

**Comment by djmitche** [comment]
r+ :)

File: test/integration/allow_ptrace_test.js (hunk 1 above)
Comment (gregarndt) [inline test/integration/allow_ptrace_test.js]: Yea, with newer versions of docker testing ptrace in a docker in docker container no longer works, but it does test in a vm just fine.  I removed it here to hopefully reduce noise in the tests for wander's review, but well, that didn't work out either too well.
